src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -0,0 +1,188 @@
+import { SearchResults } from 'algoliasearch-helper';
+import escapeHits, { TAG_PLACEHOLDER } from '../../lib/escape-highlight';
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+  noop,
+  warning,
+} from '../../lib/utils';
+import {
+  RendererOptions,
+  Renderer,
+  WidgetFactory,
+  Hits,
+  InstantSearch,
+  Unmounter,
+} from '../../types';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'autocomplete',
+  connector: true,
+});
+
+interface AutocompleteIndex {
+  indexName: string;
+  hits: Hits;
+  results: SearchResults;
+}
+
+interface AutocompleteConnectorParams {
+  /**
+   * Escapes HTML entities from hits string values.
+   *
+   * @default `true`
+   */
+  escapeHTML?: boolean;
+}
+
+export interface AutocompleteRendererOptions<TAutocompleteWidgetParams>
+  extends RendererOptions<TAutocompleteWidgetParams> {
+  currentRefinement: string;
+  indices: AutocompleteIndex[];
```

Comment:
Do we keep `indices` or rename it to `items` like the other widgets?

Reply:
`items` refers most of the time to a list of facet values, `hits` refers to a list of hits, `indices` could make sense. Both are valid, I don't think it matters much.

Reply:
Let's keep it that way, then!